tests/test_handlers.py

```diff
@@ -82,14 +86,14 @@ def setUpClass(cls) -> None:
     # override
     def setUp(self) -> None:
         self.raw_log_path: Path = LOG_DIR / Path(f"{self.id()}.log")
-        self.clp_log_path: Path = LOG_DIR / Path(f"{self.id()}.clp.zst")
+        self.clp_log_path: Path = LOG_DIR / Path(f"{self.id()}.clp")
```

Comment:
Does something add the `.zst` in the compressed case or we're just forgoing it for testing?

Reply:
I just removed it for testing to keep things general.